src/components/universal/core/Badge/Badge.recipe.ts

```diff
@@ -0,0 +1,42 @@
+import { defineRecipe } from "@pandacss/dev";
+
+export const badgeRecipe = defineRecipe({
+  className: "badge",
+  description: "The styles for the Badge component",
+  base: {
+    borderRadius: "full",
+    fontWeight: "medium",
+    display: "inline-flex",
+    alignItems: "center",
+    userSelect: "none",
+    whiteSpace: "nowrap",
+  },
+  defaultVariants: {
```

Comment:
`defaultVariants` have given us trouble in the past. Haven't tested yet, but my recommendation would be to add these styles (from the default variants) to the `base`, and remove those variants respectively to match what we have done so far.